lib/fizzbuzz.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'pry'
+class FizzBuzz
+
+  def initialize(number)
+    @number =  number
+  end
```

Comment:
The way you have done it is fine. It works as expected. I personally would pass the number in to the method, rather than where you initialise the class. You could then just make the methods class methods rather than instance methods.

I would do that because the methods are quite simple, they are doing a check and returning a thing. There isn't any changing state. For example the `@number` variable never changes once its initialised, so you could just pass it in to the method.

Reply:
I have added the changes @kks110 thank you for your feedback!